artview/components/nav.py

```diff
@@ -84,6 +83,9 @@ def __init__(self, Vradar=None, Vgrid=None, Vfilelist=None,
 
         self.layout.addLayout(self.generalLayout, 0, 0, 1, 2)
 
+        self.NewFile(self.Vradar, None, True)
+        self.NewFilelist(self.Vfilelist, None, True)
+
```

Comment:
to avoid making "compatibility" test in the creation of "display", I just create them empty and force an update. This has no significant overhead!